helix-rest/src/test/java/org/apache/helix/rest/metadatastore/accessor/TestZkRoutingDataReader.java

```diff
@@ -104,11 +104,10 @@ public void testGetRoutingDataMissingMSRDChildren() {
     _baseAccessor.create(MetadataStoreRoutingConstants.ROUTING_DATA_PATH, new ZNRecord("test"),
         AccessOption.PERSISTENT);
     try {
-      _zkRoutingDataReader.getRoutingData();
-      Assert.fail("Expecting InvalidRoutingDataException");
+      Map<String, List<String>> routingData = _zkRoutingDataReader.getRoutingData();
```

Comment:
Do we need to add tests for the newly supported methods?

Reply:
Yes. I realized that the constructor was not populating `_routingDataMap`. That has been fixed.